overlore/config.py

```diff
@@ -50,14 +50,20 @@ def load(self) -> None:
         dotenv_path = ".env.production" if self.prod is True else ".env.development"
         load_dotenv(dotenv_path=dotenv_path)
 
-        self.TORII_WS = str(os.environ.get("TORII_WS"))
-        self.KATANA_URL = str(os.environ.get("KATANA_URL"))
-        self.TORII_GRAPHQL = str(os.environ.get("TORII_GRAPHQL"))
+        tmp_TORII_WS = os.environ.get("TORII_WS")
```

Comment:
lowercase everything, upper case is for "global constants" usually

Reply:
Noted, and done